quarkus-extension/engine/runtime/src/main/java/org/camunda/bpm/quarkus/engine/extension/impl/CamundaEngineRecorder.java

```diff
@@ -55,6 +55,10 @@ public void configureProcessEngineCdiBeans(BeanContainer beanContainer) {
                                                                                        CamundaEngineConfig config) {
     QuarkusProcessEngineConfiguration configuration = beanContainer.instance(QuarkusProcessEngineConfiguration.class);
 
+    // apply properties from config before the datasource configuration.
+    // We always want to use a Quarkus datasource.
```

Comment:
Why is this only relevant for the data source and not for the other configurations?

Reply:
My point was to ensure that the line below is not moved after the datasource config. But I generalized it now, so it will still be correct.